docs/indepth/installation.rst

```diff
@@ -324,6 +328,72 @@ Setup the pre-requisite repositories and install the ``clusterhq-flocker-node``
 .. task:: install_flocker ubuntu-14.04
    :prompt: [root@ubuntu]#
 
+.. _authentication:
+
+Cluster Authentication Layer Configuration
+------------------------------------------
+
+Communication between the different parts of your cluster is secured and authenticated via TLS.
+The Flocker CLI package includes the ``flocker-ca`` tool that is used to generate TLS certificate and key files that you will need to copy over to your nodes.
+
+Once you have installed the ``flocker-node`` package, you will need to generate:
+
+- A control service certificate and key file, to be copied over to the node running your :doc:`control service <../advanced/architecture>`.
```

Comment:
The terminology is a bit confusing here, especially insofar as we want to recommend running the control service on different machine than the nodes. So file follow-up issue for documentating that recommendation.